### PR TITLE
VZ-6418. Remove Jenkinsfile references to install/uninstall jobs

### PR DESCRIPTION
**Pull request by michael-cico**
Remove Jenkinsfile hooks to grab install/uninstall job logs.

### CODE REVIEW

File: platform-operator/controllers/verrazzano/uninstall.go [hunk 1]
```diff
@@ -289,7 +289,7 @@ func (r *Reconciler) uninstallCleanup(ctx spi.ComponentContext) (ctrl.Result, er
 	}
 
 	if err := r.nodeExporterCleanup(ctx.Log()); err != nil {
-		return ctrl.Result{}, err
+		return err
```
Comment (pfmackin) [inline platform-operator/controllers/verrazzano/uninstall.go:292]: should also return ctrl.Result

Comment (michael-cico) [reply platform-operator/controllers/verrazzano/uninstall.go:292]: this looks like a botched rebase.

Comment (bobdonat) [reply platform-operator/controllers/verrazzano/uninstall.go:292]: does this compile

Comment (michael-cico) [reply platform-operator/controllers/verrazzano/uninstall.go]: Fixed.

**Pull request merged**